packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2784,16 +2787,61 @@ module.exports = function (log, config, bounces) {
     // set this to avoid passing `metricsEnabled` around to all link functions
     this.metricsEnabled = metricsEnabled;
 
+    const localizedUrls = (message) => {
+      const defaultSureyUrl =
+        'https://survey.alchemer.com/s3/6534408/Privacy-Security-Product-Cancellation-of-Service-Q4-21';
+
+      const urls = {};
+
+      if (
+        config.subscriptions.productConfigsFirestore.enabled &&
+        message.planConfig
+      ) {
+        // we are not using `determineLocale` because the product config might support more locales than the FxA supported locales list
+        const locales = message.acceptLanguage ? [message.acceptLanguage] : [];
+        const localizedConfigs = localizedPlanConfig(
```

Comment:
When I tested this locally, for a valid locale, I wasn't getting the expected localizedConfigs back.

After some investigating, I found that `getMergedConfig` wasn't working cause `this.products` didn't have the data from Firestore in it, as expected.

This seems to be related to the `select()` in manager.ts `load` [method](https://github.com/mozilla/fxa/blob/380b990379e91ea24089ded224c8bb4c9a6cead4/packages/fxa-shared/payments/configuration/manager.ts#L77).

Once I removed the `select()` it worked as expected. Also just FYI, I tested this with Meghan last week, and she had the same result as me.